jetstream/analysis.py

```diff
@@ -120,17 +122,32 @@ def _get_timelimits_if_ready(
             **time_limits_args,
         )
 
-    def _table_name(self, window_period: str, window_index: int) -> str:
+    def _table_name(
+        self, window_period: str, window_index: int, analysis_basis: Optional[AnalysisBasis] = None
+    ) -> str:
         assert self.config.experiment.normandy_slug is not None
```

Comment:
might be worth a docstring here explaining that aggregates tables should have an AnalysisBasis but statistics tables shouldn't